src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,71 @@
+package lotto.model;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Lotto {
+
+    private final List<Integer> numbers;
```

Comment:
원시 값을 포장해 로또 번호 범위에 대한 책임을 위임하면 어떨까요?
```suggestion
    private final List<LottoNumber> numbers;
```

Reply:
@testrace 
`LottoNumber` 를 생성했습니다.